### PR TITLE
Fix: safe alternative to uuid.New() and uuid.NewString() functions

### PR DESCRIPTION
**Pull request by guacamole**
- This PR fixes issue #125 
- Replaced uuid functions with uuid.NewRandom which returns an error along with a random uuid
Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: router/router.go [hunk 1]
```diff
@@ -35,7 +36,10 @@ func Register(
 
 	e.Use(middleware.RequestIDWithConfig(middleware.RequestIDConfig{
 		Generator: func() string {
-			requestId := uuid.New()
+			requestId, err := uuid.NewRandom()
+			if err != nil {
+				time.Now().Format(time.RFC3339Nano)
```
Comment (jay-dee7) [inline router/router.go:41]: haha we need to set the id variable here @guacamole

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
@guacamole this PR is mostly good, except one missed var assignment in `RequestIDMiddleware`, everything else is perfect 🥇

File: router/router.go (hunk 1 above)
Comment (guacamole) [reply router/router.go]: I totally missed to add a return there jeez

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 🥇 🥇

**Pull request merged**